GoogleForCreators/web-stories-wp
# Preset style improvements.

## Summary

Improves preset style:
- Adds padding where necessary
- Removes empty space from Page/Shape presets


## Relevant Technical Choices

Uses correct count for presets when deciding if there are presets present for that specific element. This count is used for deciding if to use panel padding and for the default height of the panel.


## To-do



## User-facing changes

- There is now padding in the Preset Panel even if only Style Presets are present.


## Testing Instructions

1. Add a Text element.
2. Remove all color presets.
3. Add a Style Preset by setting the Background to "Fill" and then clicking "+" icon in the preset panel
4. Verify there the preset panel has padding and the Style Presets are not next to the left edge.



---



Fixes #1854
